### DISCUSSION TITLE
Hrule equivalent

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#4279] [Q&A] [answered]
Hi guys,

I'd like to know if there is a clean way to mimic `hrule` in typst ?

I want to trace a line as a separator without specifying length, ...

Thanks :smile:

**Comment by jamesrswift** [comment, marked as answer]
`line(width: 100%)`. You probably want to style the stroke a bit to get an output you're more used to: `line(width: 100%, stroke: 0.75pt)`.

**Reply by trya2l** [reply]
I didn't thought this would work. Thanks!

**Comment by brianclinkenbeard** [comment]
By default, `\hrule` has a height of 0.4pt.

```typ
#let hrule = line(width: 100%, stroke: 0.4pt)
#hrule
```
If you want the same for hlines in tables and grids:
```typ
#set table.hline(stroke: 0.4pt)
#set grid.hline(stroke: 0.4pt)
```
